src/main/java/seedu/address/autocomplete/PrefixWord.java

```diff
@@ -1,17 +1,24 @@
 package seedu.address.autocomplete;
 
+import java.util.LinkedList;
+
 /**
  * Represents a prefix word(eg: n/, t/) in application
  */
-public class PrefixWord extends AutoCompleteWord {
-    private String associatedCommandWord;
+public class PrefixWord extends AutoCompleteWord implements AssociableWord {
+    private String associatedWord1;
```

Comment:
Why got 2 words?

Reply:
Need to have 2 associated words because the prefix that will be suggested depends on both object word and command word. Eg like "pat-edit" and "appt-edit" will have different sets of prefix.